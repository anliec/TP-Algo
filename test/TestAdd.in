ADD 666 2015 11 11 11 1 1 V
STATS_C 666
ADD 666 2015 11 11 11 2 1 J
STATS_C 666
ADD 666 2015 11 11 11 3 1 R
STATS_C 666
ADD 666 2015 11 11 11 4 1 N
STATS_C 666
ADD 666 2015 11 11 11 5 1 V
STATS_C 666
ADD 666 2015 11 11 11 6 1 J
STATS_C 666 1 20 26

EXIT