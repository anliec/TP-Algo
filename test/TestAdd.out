V 100%
J 0%
R 0%
N 0%
V 0%
J 100%
R 0%
N 0%
V 0%
J 0%
R 100%
N 0%
V 0%
J 0%
R 0%
N 100%
V 100%
J 0%
R 0%
N 0%
V 66%
J 33%
R 0%
N 0%